openslides/motions/static/templates/motions/motion-form.html

```diff
@@ -29,15 +33,17 @@ <h1 ng-if="!motion.id" translate>New motion</h1>
         <div ng-bind-html="user.get_short_name() | highlight: $select.search"></div>
         <small ng-bind-html="user.structure_level | highlight: $select.search"></small>
       </ui-select-choices>
-    </ui-select>
+    </ui-select>-->
   </div>
-  <div class="form-group" ng-class="{'has-error has-feedback': motionForm.inputTitle.$error.required}">
+  <div class="form-group">
     <label for="inputName" translate>Title</label>
-    <input type="text" ng-model="motion.title" class="form-control" name="inputTitle" ng-required="true">
+    <!-- TODO: get title from latest version for edit form (ng-value does not work) -->
+    <input type="text" ng-model="motion.title" ng-value="motion.versions.slice(-1)[0].title" class="form-control" name="inputTitle" required>
```

Comment:
@ostcar Here I need your help. In the motion-edit view the title and text fields are empty. How can I show the values of motion.versions without changing the ng-model (which is required for setting the form data).